tests/virt_autotest/virt_utils.pm

```diff
@@ -143,5 +143,23 @@ sub update_guest_configurations_with_daily_build {
     # repl_guest_autoyast_addon_with_daily_build_module;
 }
 
+sub clean_up_red_disks {
+    my $wait_script           = "120";
+    my $get_disks_not_used    = "ls /dev/sd* | grep -v -e \"/dev/sd[a].*\" -e \"/dev/sd[b-z].*[[:digit:]].*\"";
+    my $disks_not_used        = script_output($get_disks_not_used, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $get_disks_fs_overview = "lsblk -f";
+    my $disks_fs_overview     = script_output($get_disks_fs_overview, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $get_fs_type_supported = "$get_disks_fs_overview | grep sda | awk \'{print \$2}\' | grep -v swap | tail -1";
+    my $fs_type_supported     = script_output($get_fs_type_supported, $wait_script, type_command => 1, proceed_on_failure => 1);
+    my $make_fs_cmd           = "mkfs.$fs_type_supported";
+
+    if (($disks_not_used ne '') && ($fs_type_supported ne '')) {
+        assert_script_run("wipefs -a $disks_not_used && $make_fs_cmd $disks_not_used");
+    }
+    else {
+        diag("Maybe there are no other disks functioning other than /dev/sda. Or something unexpected happened during obtaining available file system type.\n Disks and File Systems Overview:\n $disks_fs_overview \n");
```

Comment:
To be honest I would log something even we parsed disks, to see what we have parsed, to make it easier ti investigate what actually happened if e.g. script_output provided something which was not expected.